app/controllers/courses_controller.rb

```diff
@@ -15,7 +15,9 @@ class CoursesController < ApplicationController
   def create
     slug_from_params if should_set_slug?
     @course =
-      Course.create(course_params.merge('passcode' => Course.generate_passcode))
+      Course.create(course_params.merge(
+        passcode: Course.generate_passcode,
+        type: ENV['default_course_type']))
```

Comment:
Okay, I see now that setting `type: nil` actually sets to null rather than the db default.  I don't like repeating the hardcoded default in multiple places, though.

What if we create a file for default configuration instead, and leave the setting commented in application.example.yml to make it clear that it's an optional value?